Bottom: 619b5d90634d9c72d8bb012f8a39ff7ae5cadc34
Top:    f7b7c0ba51ba945afa89f9b729e820e0e70d11b6
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-02-01 16:20:07 -0600

Fix .ssh folder permission

The start.sh scripts were mistakenly changing the permissions of
$HOME/.ssh to 0600, removing the execute bit for the user and 
effectively removing access.


---
diff --git a/docker/buildbot-master/buildbot/start.sh b/docker/buildbot-master/buildbot/start.sh
index 63b884d..a30fbf6 100755
--- a/docker/buildbot-master/buildbot/start.sh
+++ b/docker/buildbot-master/buildbot/start.sh
@@ -2,7 +2,7 @@
 
 mkdir -p $HOME/.ssh
 cp -R /run/secrets/ssh/* $HOME/.ssh
-chmod -R 600 $HOME/.ssh
+chmod -R 600 $HOME/.ssh/*
 
 ln -v -s -f /home/buildbot/buildbot.tac /home/buildbot/master.cfg /home/buildbot/*.py /buildbot
